google-cloud-storage/lib/google/cloud/storage/service.rb

```diff
@@ -656,6 +656,7 @@ def delete_file_acl bucket_name, file_name, entity, generation: nil,
         # Returns Google::Apis::StorageV1::HmacKey.
         def create_hmac_key service_account_email, project_id: nil,
                             user_project: nil, options: {}
+          options[:retries] = 0
```

Comment:
Can we make it configurable? Give users option to make it retryable if they want. You can refer this [code](https://github.com/googleapis/google-cloud-ruby/pull/18834/files#diff-7b94a6bb6adeaa1a8d4052dc0895cb31ff917367272c55d49f8e21c87db78c5aR138) for the same.

Reply:
By default, it'll be non retryable. But users should have option to override it.

Reply:
I believe there's one more API which is non idempotent. Can we also include that?